lib/validation.js

```diff
@@ -209,43 +209,36 @@ module.exports = function validation (yargs, usage, y18n) {
     return implied
   }
 
+  function coerceValueType (argv, val) {
```

Comment:
I suggest renaming this function, as it is not coercing the value type, but checking its presence.

```suggestion
  function keyExists (argv, key) {
```